### ISSUE TITLE
Fix settings scrollview layout

### ISSUE THREAD

**Issue by amulware** [toggl/mobileapp#1688]
There is definitely a glitch on iPhoneX where the scrollview does not reach all the way to the bottom of the screen (you have to drag down the scrollview to see this):

<img width="288" alt="screen shot 2018-03-08 at 15 50 05" src="https://user-images.githubusercontent.com/3987892/37162074-3d417b26-22ec-11e8-912d-59f97556578f.png">

We should also look into how it looks on iPhone SE, where if scrolled all the way down, the signout button touches the bottom of the screen, while on larger phones it does not. Ping @pe0ny for this one.

**Comment by pe0ny** [comment]
Yep, the Sign out button shouldn't touch the bottom of the screen. Padding between the Sign out button and the bottom of the screen should always be 24pt like shown in here: https://zpl.io/29EBEz2
